test/README.md

```diff
@@ -75,7 +77,7 @@ To cleanup all tests separately, use the following command from test/:
 
 ## Adding a new recipe test
 
-For a new recipe, in addition to its yaml file and REAME.md, it should also include a set of test files to make sure the recipe is functional and up-to-date.
+For a new recipe, in addition to its yaml file and REAME.md, it should also include a set of test files to make sure the recipe is functional and up-to-date. In the description section of the pull request, you should also provide the result of `make test` tp show your test is passing and is not breaking any other existing tests. See example in [output-example.txt](./test_example/output_example.txt). 
```

Comment:
tp show => to show

any other existing => other tests

Reply:
Updated.